Makefile

```diff
@@ -48,11 +48,6 @@ PYTHON_PATH_SDK = PYTHONPATH=/application-services/megazord
 
 JETSTREAM_CONFIG_URL = https://github.com/mozilla/metric-hub/archive/main.zip
 
-FIREFOX_ANDROID_REPO = @mozilla-mobile/firefox-android
-FIREFOX_IOS_REPO     = @mozilla-mobile/firefox-ios
-FOCUS_IOS_REPO       = @mozilla-mobile/focus-ios
-MONITOR_REPO         = @mozilla/blurts-server
```

Comment:
Why are these being removed in this PR? Can they be removed separately?

Reply:
oh so they are not being used, suggested to remove - https://github.com/mozilla/experimenter/pull/9970#discussion_r1432091055

Reply:
Ahh, perfect! 👍